package.json

```diff
@@ -36,10 +36,9 @@
   },
   "scripts": {
     "test": "gulp",
-    "postinstall": "yodoctor && npm run tabtab",
```

Comment:
Ideally, I'd like a way to setup auto completion as part of the post install. But that process needs to be transparent to user.

Ideally, that process wouldn't prompt. It would just default to best choice or skip otherwise.

Then running `yo completion` would prompt for choices if we cannot figure out the default ways.

What do you think?